### PR TITLE
[3.10] Remove FLoC

### PR DESCRIPTION
**Pull request by brianteeman**
backport of #36855  .

### CODE REVIEW

**Comment by zero-24** [comment]
@richard67 didnt we also have to comment (not remove) the old update sqls which added the postinstall? Or would than in this case the new SQl fail to execute?

**Comment by zero-24** [comment]
FYI as 4.1.0 is in RC and language freeze mode which does not allow language changes and technically this is a language change as the strings are removed as well as we usually only try to do non-critical bug fixes. This removal change will go into 4.1.1 and 3.10.7 as coordinated with @bembelimen 
As there is also no major impact of the header present or not this does not have to go into the very next release anyway.

**Comment by brianteeman** [comment]
Please dont pretend that it is anything to do with a language string freeze. 

Obviously removing a string has zero impact on a translation.

And of course you're quite happy to merge other pull requests that actually change a language string

**Comment by zero-24** [comment]
> And of course you're quite happy to merge other pull requests that actually change a language string

The last merges have been coordinated between benjamin, me and the TTs. But we should always try to stick to the rules we apply on ourself. Yes we are not perfect but there is no change that has an impact in the real world here so I dont think we have to ship that with the next possible release in this special case.

And as mentiond in my message its not just the language string thing ;)

**Comment by richard67** [comment]
> @richard67 didnt we also have to comment (not remove) the old update sqls which added the postinstall? Or would than in this case the new SQl fail to execute?

Why should the new SQL fail? When there is no record which matches the where clause, it will simply not do anything, but it will not fail.

Changing the old SQL is not necessary. Personally I would do it because in my opinion it is silly to insert something just to remove it later, and we safely could change them. But some people always make a big yelling when we change old update SQL, and because it’s not really necessary and to avoid discussions I would say leave the old SQL as they are.

**Comment by zero-24** [comment]
Thanks for checking 👍

File: installation/sql/mysql/joomla.sql [hunk 1]
```diff
@@ -1677,7 +1677,6 @@ INSERT INTO `#__postinstall_messages` (`extension_id`, `title_key`, `description
 (700, 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_TITLE', 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_BODY', 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_ACTION', 'plg_captcha_recaptcha', 1, 'action', 'site://plugins/captcha/recaptcha/postinstall/actions.php', 'recaptcha_postinstall_action', 'site://plugins/captcha/recaptcha/postinstall/actions.php', 'recaptcha_postinstall_condition', '3.8.6', 1),
 (700, 'COM_ACTIONLOGS_POSTINSTALL_TITLE', 'COM_ACTIONLOGS_POSTINSTALL_BODY', '', 'com_actionlogs', 1, 'message', '', '', '', '', '3.9.0', 1),
 (700, 'COM_PRIVACY_POSTINSTALL_TITLE', 'COM_PRIVACY_POSTINSTALL_BODY', '', 'com_privacy', 1, 'message', '', '', '', '', '3.9.0', 1),
```
Comment (Quy) [inline installation/sql/mysql/joomla.sql]:
```suggestion
(700, 'COM_PRIVACY_POSTINSTALL_TITLE', 'COM_PRIVACY_POSTINSTALL_BODY', '', 'com_privacy', 1, 'message', '', '', '', '', '3.9.0', 1);
```
Please fix in the other files.

Comment (brianteeman) [reply installation/sql/mysql/joomla.sql]: grrh

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 36faabb9f3dc1aa00195893cae2e9b1acfb11383<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36861">issues.joomla.org/tracker/joomla-cms/36861</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 36faabb9f3dc1aa00195893cae2e9b1acfb11383<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36861">issues.joomla.org/tracker/joomla-cms/36861</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36861">issues.joomla.org/tracker/joomla-cms/36861</a>.</sub>

**Comment by zero-24** [comment]
Merging 🎉

**Comment by brianteeman** [comment]
thx

**Pull request merged**